### PR TITLE
Updated the java, tomcat, nodejs and php packages to recent versions

### PR DESCRIPTION
**Pull request by yaso195**
This commit resolves ENGT-4615.  @variadico @zquestz

### CODE REVIEW

File: runtimes/php-fpm-nginx-5.6.16.conf [hunk 1]
```diff
@@ -0,0 +1,157 @@
+name:      "php-fpm-nginx-5.6.16"
+namespace: "/apcera/pkg/runtimes"
+
+# -apc4 for libpng 1.6.19 security update
+# -apc1 for freetype-2.5.3 security update
```
Comment (variadico) [inline runtimes/php-fpm-nginx-5.6.16.conf]: There are no `php-fpm-nginx-5.6.16-apc4` or `php-fpm-nginx-5.6.16-apc1`. These comments are incorrect.

Comment (variadico) [inline runtimes/php-fpm-nginx-5.6.16.conf]: Also, @zquestz said something about adding his changes. Not sure if this new version incorporates his changes or not.

Comment (zquestz) [inline runtimes/php-fpm-nginx-5.6.16.conf]: It has the changes!

**Comment by variadico** [comment]
LGTM

File: runtimes/php-fpm-nginx-5.6.16.conf (hunk 1 above)
Comment (yaso195) [inline runtimes/php-fpm-nginx-5.6.16.conf]: Done

**Pull request merged**